app/src/main/java/com/android/consumerfinancehalan/common/mapper/Mapper.kt

```diff
@@ -0,0 +1,5 @@
+package com.android.consumerfinancehalan.common.mapper
+
+interface Mapper<F,T> {
+    fun mapTo(from: F): T
+}
```

Comment:
why you decided to have an interface for a mapper

Reply:
This offers code reusability by allowing the same mapping logic to be used in multiple places and reducing duplicate code. It enables the introduction of new mapping scenarios without changing existing code that relies on the mappers.